docs/src/main/tut/json.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: default
+title: JSON handling
+---
+
+## Pick a library
+
+### Argonaut
+
+```scala
+libraryDependencies += Seq(
+  "org.http4s" %% "http4s-argonaut" % "0.15.0a-SNAPSHOT",
+  "com.github.alexarchambault" %% "argonaut-shapeless_6.1" % "1.1.1" // or 1.1.0 for non-a versions
+)
+```
+
+### Circe
+
+```scala
+libraryDependencies += "org.http4s" %% "http4s-circe" % "0.15.0a-SNAPSHOT"
+```
+
+### Json4s
+
+```scala
+libraryDependencies += "org.http4s" %% "http4s-json4s" % "0.15.0a-SNAPSHOT"
+```
+
+And one of
+
+```scala
+libraryDependencies += "org.http4s" %% "http4s-json4s-native" % "0.15.0a-SNAPSHOT"
+libraryDependencies += "org.http4s" %% "http4s-json4s-jackson" % "0.15.0a-SNAPSHOT"
+```
+
+## Import it
+
+The namespace is
+
+```scala
+import org.http4s.<libraryName>._
+```
+
+where `libraryName` can be one of the above, e.g. `org.http4s.argonaut._`.
```

Comment:
Maybe lay them out explicitly: `argonaut`, `circe`, `json4s.native`, `json4s.jackson`.  New Scala users might believe `<libraryName>` is valid syntax.  I've learned people copy and paste the syntax-highlighted bits first and read second.